src/Appwrite/Platform/Modules/Functions/Http/Executions/Create.php

```diff
@@ -221,13 +221,16 @@ public function action(
             'scopes' => $function->getAttribute('scopes', [])
         ]);
 
+        $executionId = ID::unique();
+        $headers['x-appwrite-execution-id'] = $executionId ?? '';
         $headers['x-appwrite-key'] = API_KEY_DYNAMIC . '_' . $apiKey;
         $headers['x-appwrite-trigger'] = 'http';
         $headers['x-appwrite-user-id'] = $user->getId() ?? '';
         $headers['x-appwrite-user-jwt'] = $jwt ?? '';
         $headers['x-appwrite-country-code'] = '';
         $headers['x-appwrite-continent-code'] = '';
         $headers['x-appwrite-continent-eu'] = 'false';
+        $headers['x-appwrite-client-ip'] = $request->getIP();
 
         $ip = $headers['x-real-ip'] ?? '';
```

Comment:
Hmm this seems wrong now, I think we should use $request->getIP() here too.

Let's check where else we use this x-real-ip directly, and consider replacing it there as well